### ISSUE TITLE
Fedora 28: Unable to attach to CoreCLR

### ISSUE THREAD

**Issue by HenrikBach1** [OmniSharp/omnisharp-vscode#2665]
## Environment data
`dotnet --info` output:
```bash
$ dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   2.1.403
 Commit:    04e15494b6

Runtime Environment:
 OS Name:     fedora
 OS Version:  28
 OS Platform: Linux
 RID:         fedora.28-x64
 Base Path:   /usr/share/dotnet/sdk/2.1.403/

Host (useful for support):
  Version: 2.1.5
  Commit:  290303f510

.NET Core SDKs installed:
  2.1.403 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.5 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.5 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.5 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```
VS Code version:
```
Version: 1.25.1
Commit: 1dfc5e557209371715f655691b1235b6b26a06be
Date: 2018-07-11T15:40:20.190Z
Electron: 1.7.12
Chrome: 58.0.3029.110
Node.js: 7.9.0
V8: 5.8.283.38
Architecture: x64
```

C# Extension version:

```
1.15.2
```

## Steps to reproduce

I follow this document to build a sample dotnet project: `https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial`. Then I set a break point at `Console.WriteLine("Hello World!");`.

## Expected  behavior

Debugging :)

## Actual behavior

Error message:

```
-------------------------------------------------------------------
You may only use the Microsoft .NET Core Debugger (vsdbg) with
Visual Studio Code, Visual Studio or Visual Studio for Mac software
to help you develop and test your applications.
-------------------------------------------------------------------
Unable to attach to CoreCLR.
```

Help is appreciated.

Thank you.

/Henrik

**Comment by walkjivefly** [comment]
@HenrikBach1 Try disabling razor! Take a look at [this](https://github.com/aspnet/Razor.VSCode/issues/223)

Go to FIle/Preferences/Settings, search for razor and set Razor:Disabled. 
Restart. 
Voila! 
Debugging :-)

ymmv but it works for me.

**Comment by HenrikBach1** [comment]
Hi,

Thank you for your suggestions. I'll look into them in a moment...

/Henrik

**Comment by HenrikBach1** [comment]
Hi,

Neither in user `space settings`, nor `workspace settings` is this `"razor.disabled"` property recognized. 

And, Neither is it when searching for razor in: `Try a preview of our >new settings editor<`.

/Henrik

**Comment by walkjivefly** [comment]
Debugging works for me with Razor 1.17.1 installed and without it being disabled.

But @HenrikBach1 In VSCode File/Preferences/Settings gets me here:
![image](https://user-images.githubusercontent.com/17745270/49271431-3111e080-f4a0-11e8-8346-fe3d42edcd9c.png)

**Comment by walkjivefly** [comment]
@HenrikBach1 Sorry, I don't know what else to say. On the menubar I click File, then Preferences, then settings (or just press Ctrl comma) and I get the panel shown in my screenshot above.
I only installed VSCode to be able to (try to) debug NBitcoin. I've never used DotNet or C# before, probably never will again once I've done the NBitcoin stuff I want to do.
I doubt it's relevant but I installed the following extensions:
![image](https://user-images.githubusercontent.com/17745270/49328137-e080ad00-f59d-11e8-8a49-9989d006b697.png)
